src/index.js

```diff
@@ -1,15 +1,22 @@
 import React from "react";
 import ReactDOM from "react-dom";
-import "./index.css";
+import {BrowserRouter} from "react-router-dom"
+import { Provider } from "react-redux";
 import App from "./App";
 import { makeServer } from "./server";
+import store  from "./Redux/store"
 
 // Call make Server
 makeServer();
 
 ReactDOM.render(
   <React.StrictMode>
+    <Provider store={store}>
+    <BrowserRouter>
+
     <App />
+    </BrowserRouter>

```

Comment:
Suggestion please keep something in your vscode so that you can indent your code properly